server/conf/application.conf

```diff
@@ -308,6 +308,28 @@ play.filters {
   enabled += filters.LoggingFilter
   enabled += filters.ValidAccountFilter
 
+  # GZip filter configuration
+  gzip {
```

Comment:
I'd consider removing these settings (as they're presumably pretty similar to the default behavior we'd get out of the box). In my opinion, the main savings will come from having any gzip at all. If we find the defaults to be unacceptable, we can tweak them at that point.

Reply:
Yes, please remove these